hack/bootstrap-kubestellar.sh

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The KCP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: $0  --verbose
```

Comment:
What is this line for?  It omits almost all --- but not all --- of the command line options.

Reply:
I'm not sure, in some cases I saw it used as an example/default usage. I'll be happy yo change it to what you think is best. It does not matter for me

Reply:
I think it is confusing to label _typical_ usage as simply "usage".  Changing the heading to _Typical usage_ would make your intent clear.